### PR TITLE
The Lord of the Rings (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
brave_VNlb9YOVl7
brave_qOL6fUd4yj

---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/The Lord of the Rings/readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+List of all movie quotes.
+### Request a quote
+Request one specific movie quote.
+### List chapters
+List of all book chapters
+### Request a chapter
+Request one specific book chapter.
+
+## Known Issues and Limitations
+There are no known issues at this time.
```
[the remark this answers is not in the document]
Comment (troystaylor) [reply independent-publisher-connectors/The Lord of the Rings/readme.md:41]: Optional

File: independent-publisher-connectors/The Lord of the Rings/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1635 @@
...
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/book": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```
Comment (v-eisljami) [inline independent-publisher-connectors/The Lord of the Rings/apiDefinition.swagger.json:25]: Please do not use default responses. Add description for responses. Everything else seems to be in order. Looking forward to certifying your connector.

Comment (v-eisljami) [inline independent-publisher-connectors/The Lord of the Rings/apiDefinition.swagger.json:25]: Please do not use default responses. Add description for responses. Everything else seems to be in order. Looking forward to certifying your connector.

**Review by v-eisljami — CHANGES_REQUESTED** [review]

**Review by v-eisljami — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@troystaylor Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**